### PR TITLE
Add support for `Composition::$hideDefaultOnly` option, bc-safe

### PR DESCRIPTION
**Pull request by Webkadabra**
https://github.com/luyadev/luya-core/pull/5

### CODE REVIEW

File: src/widgets/LangSwitcher.php [hunk 1]
```diff
@@ -184,10 +184,15 @@ public function run()
 
         $items = [];
 
+        if (!Yii::$app->composition->hidden && property_exists(Yii::$app->composition, 'hideDefaultOnly')) {
+            $hideDefaultOnly = Yii::$app->composition->hideDefaultOnly;
+            Yii::$app->composition->hideDefaultOnly = false;
```
Comment (nadar) [inline src/widgets/LangSwitcher.php]: Why do have to set the property to false at this point?

Comment (Webkadabra) [reply src/widgets/LangSwitcher.php]: if it's not off then widget will not add prefixes to links in switch widget.

Comment (Webkadabra) [reply src/widgets/LangSwitcher.php]: default language would still have correct, non-prefixed, link.

**Comment by nadar** [comment]
I think this is fixed in the luya core, if the option is enabled to url should be generated correctly, otherwise we need a unit test.

**Pull request closed without merging**